### DISCUSSION TITLE
magick: webp2gif,-delay 12.8  Invalid setting

### DISCUSSION THREAD

**Discussion by ansarx** [ImageMagick/ImageMagick#7540] [Development]
### ImageMagick version

7.1.1-Q16

### Operating system, version and so on

windows 10

### Description

magick -delay 12.8 test.webp test.gif

webp is 8 frames, but the resulting test.gif is 30 frames. Is there any way to control the frame rate? I want to maintain 8 frames

**Comment by fmw42** [comment]
The does not make sense. Imagemagick will not increase your number of frames.  Post your webp file. Perhaps it has multiple loops or multiple pages per image. If you cannot post the file, then show the results from `magick identify -verbose test.webp`.  How many images does that show?
